src/Patch/Cli/Command/ApplyCommand.php

```diff
@@ -109,6 +164,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $config = $this->getConfiguration();
 
+        $this->checkPhpConstraint($config);
```

Comment:
Worth a comment to explain why this should be ran here rather than in the patch strategy with other checks? I'm thinking it should be here so it's run before any scripts or locking for patching :+1: